### PR TITLE
Add lp token rpc

### PR DESCRIPTION
**Pull request by benluelo**
## Issue
https://app.clickup.com/t/2f75nqq

## Checklist

- ~~[ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~~
- ~~[ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~~

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2f75nqq RPC to return the amount of LP tokens](https://app.clickup.com/t/2f75nqq)

**Review by hussein-aitlahcen — APPROVED** [review]

File: frame/pablo/src/stable_swap.rs [hunk 1]
```diff
@@ -253,4 +200,84 @@ impl<T: Config> StableSwap<T> {
 		T::Assets::burn_from(pool.lp_token, who, lp_amount)?;
 		Ok((base_amount, quote_amount, total_issuance))
 	}
+
+	pub(crate) fn calculate_mint_amount_and_fees(
```
Comment (dzmitry-lahoda) [inline frame/pablo/src/stable_swap.rs:204]: assume this is cp

File: node/src/runtime.rs [hunk 2]
```diff
@@ -192,17 +192,9 @@ define_trait! {
...
-			}
-		}
+		impl for composable_runtime {}
 
-		impl for picasso_runtime {
-			fn (io, deps) {
-				io.extend_with(PabloApi::to_delegate(Pablo::new(deps.client)));
-			}
-		}
+		impl for picasso_runtime {}
```
Comment (dzmitry-lahoda) [inline node/src/runtime.rs:197]: hm, not sure, should if be feature gates?

**Review by dzmitry-lahoda — APPROVED** [review]

File: runtime/picasso/src/lib.rs [hunk 3]
```diff
@@ -942,23 +942,6 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl pablo_runtime_api::PabloRuntimeApi<Block, PoolId, CurrencyId, Balance> for Runtime {
```
Comment (vimukthi-git) [inline runtime/picasso/src/lib.rs:945]: :+1:

File: frame/pablo/rpc/src/lib.rs [hunk 4]
```diff
@@ -28,6 +28,15 @@ where
 	) -> RpcResult<
 		PriceAggregate<SafeRpcWrapper<PoolId>, SafeRpcWrapper<AssetId>, SafeRpcWrapper<Balance>>,
 	>;
+
+	#[rpc(name = "pablo_lpTokensReceivedForLiquidityProvided")]
```
Comment (vimukthi-git) [inline frame/pablo/rpc/src/lib.rs:32]: Nit:
```suggestion
	#[rpc(name = "pablo_expectedLpTokensGivenLiquidity")]
```

**Review by vimukthi-git — APPROVED** [review]
LGTM

File: frame/pablo/rpc/src/lib.rs (hunk 4 above)
Comment (dzmitry-lahoda) [reply frame/pablo/rpc/src/lib.rs:32]: would be nice making words in trait be same as in rpc ;)

Comment (benluelo) [reply frame/pablo/rpc/src/lib.rs:32]: This is a much better name, i'll rename the function as well :+1:

File: node/src/runtime.rs (hunk 2 above)
Comment (benluelo) [reply node/src/runtime.rs:197]: Not sure I follow, I removed the RPC from the picasso runtime since it was just a dummy value being returned anyways

Comment (benluelo) [reply node/src/runtime.rs:197]: (and composable as well)

**Review by vivekvpandya — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**